app/workers/github_service/closed_worker.rb

```diff
@@ -0,0 +1,11 @@
+module GithubService
+  class ClosedWorker
+    include Sidekiq::Worker
+    include PayloadParser
+
+    def perform(payload)
+      payload = parse_payload(payload)
```

Comment:
I'd rather have jobs not have logic, and just call `services`

Reply:
What do you mean? Keep the method in the `GithubHandler` service and then call it from the job? Like `GithubHandler.closed` ?

Reply:
I mean having POROs where we have the business logic and the job just calls it. for example
```
# app/jobs/github/closed_job.rb
module Github
  class ClosedJob
    ...

    def perform(payload)
       Github::ClosedService.new(payload).call
    end
  end
end

# app/services/github/closed_service.rb
module Github
  class ClosedService
    attr_reader :payload

    def initialize(payload)
      @payload = payload
    end

    def call
      # logic goes here
    end
  end
end
```

A related link about jobs: https://chriskottom.com/blog/2015/11/bulletproof-rails-background-jobs/
A related link about services and more: https://codeclimate.com/blog/7-ways-to-decompose-fat-activerecord-models/